dashboard/chart/chart.html

```diff
@@ -0,0 +1,92 @@
+<html>
+	<head>
+		<link href='https://fonts.googleapis.com/css?family=Ubuntu' rel='stylesheet' type='text/css'>
+		<link rel="stylesheet" href="chartist.css">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+		<script src="chartist.js"></script>
+		<script src="tooltip.js"></script>
+		<script src="axis.js"></script>
+		<script src="csv.js"></script>
+  </head>
+  <body>
+  
+<div class="ct-chart ct-octave" style="width: 950px;"></div>
+
+<script>
+//this is the location of the csv file
+//the format of it needs to be like this:
+//		monthyear,num_attendees
+//		2014-01,16
+//		2014-02,37
+//		2014-03,38
+var CSV_FILE = 'monthly_attendees.csv';

```

Comment:
In order to get this automatically updating, you'll have to make an ajax request to http://dxetech.org/dashboard/monthly_attendees.csv